scripts/mark_broad_excepts.py

```
@@ -0,0 +1,44 @@
+        path.write_text("\n".join(lines) + "\n", encoding="utf-8")
+    return changed
+
+
+def main():
+    changed_files = []
+    for p in sorted(Path("modul").glob("**/*.py")):
+        try:
+            if process(p):
+                changed_files.append(str(p))
+        except Exception:
+            continue
```

Ruff (0.14.13)

`try`-`except`-`continue` detected, consider logging the exception

(S112)